### PR TITLE
issue#38#39 Add Social Login from Google and Facebook

### PR DESCRIPTION
**Pull request by Jerry-1947**
I just leave console.log(user) for the reference of the response from social login

### CODE REVIEW

File: frontend/src/google.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
+  <path fill="#EA4335" d="M24 9.5c3.54 0 6.71 1.22 9.21 3.6l6.85-6.85C35.9 2.38 30.47 0 24 0 14.62 0 6.51 5.38 2.56 13.22l7.98 6.19C12.43 13.72 17.74 9.5 24 9.5z"/>
+  <path fill="#4285F4" d="M46.98 24.55c0-1.57-.15-3.09-.38-4.55H24v9.02h12.94c-.58 2.96-2.26 5.48-4.78 7.18l7.73 6c4.51-4.18 7.09-10.36 7.09-17.65z"/>
+  <path fill="#FBBC05" d="M10.53 28.59c-.48-1.45-.76-2.99-.76-4.59s.27-3.14.76-4.59l-7.98-6.19C.92 16.46 0 20.12 0 24c0 3.88.92 7.54 2.56 10.78l7.97-6.19z"/>
+  <path fill="#34A853" d="M24 48c6.48 0 11.93-2.13 15.89-5.81l-7.73-6c-2.15 1.45-4.92 2.3-8.16 2.3-6.26 0-11.57-4.22-13.47-9.91l-7.98 6.19C6.51 42.62 14.62 48 24 48z"/>
+  <path fill="none" d="M0 0h48v48H0z"/>
+</svg>
```
Comment (ashdaily) [inline frontend/src/google.svg:7]: this file should be inside `public` folder

File: frontend/src/pages/Login.js [hunk 2]
```diff
@@ -1,76 +1,131 @@
...
+						<Button variant="primary" type="submit" block>
+							Login
+						</Button>
+					</Form>
+					<center
+						style={{
+							borderBottom: "1px solid #ced4da",
+							lineHeight: "0.1em",
+                            margin: '15px 0'
+						}}
```
Comment (ashdaily) [inline frontend/src/pages/Login.js:98]: move it to scss file

File: frontend/src/pages/Login.js [hunk 3]
```diff
@@ -1,76 +1,131 @@
...
+					>
+						<span style={{ background: "#fff", padding: '0 5px' }}>OR</span>
```
Comment (ashdaily) [inline frontend/src/pages/Login.js:100]: move it to scss file

Comment (ashdaily) [inline frontend/src/pages/Login.js:100]: why not just wrap span in a p tag and use `text-center` 🤔

**Review by ashdaily — CHANGES_REQUESTED** [review]
Looks fine, just small changes.

File: frontend/src/pages/Login.js (hunk 3 above)
Comment (Jerry-1947) [reply frontend/src/pages/Login.js]: To add through line

**Review by ashdaily — APPROVED** [review]

**Pull request merged**